src/inmanta/protocol/decorators.py

```diff
@@ -183,7 +180,7 @@ def typedmethod(
 
     def wrapper(func: MethodT) -> MethodT:
         paths = path if isinstance(path, list) else [path]
-        for current_path in paths:
+        for index, current_path in enumerate(paths):
```

Comment:
This I don't understand?

Reply:
See comment about `set_method_properties_on_fnc`.